src/components/alert/Alert.stories.tsx

```diff
@@ -10,7 +10,7 @@ const meta: Meta<typeof Alert> = {
   args: {
     closable: false,
     showIcon: true,
-    banner: true,
+    banner: false,
```

Comment:
Such an odd name for a prop haha

Reply:
Actually, why is there a `closeable` and `banner` prop, I thought the banner prop was what we needed to set to false to remove the x button so now I'm confused 😂